helpers/manifestInjector.go

```diff
@@ -34,7 +34,7 @@ func InjectSteps(mft manifest.EstafetteManifest, builderTrack, gitSource string)
 		})
 	}
 
-	if !StepExists(injectedManifest.Stages, "set-pending-build-status") {
+	if !StepExists(injectedManifest.Stages, "set-pending-build-status") && gitSource != "source" {
```

Comment:
Can we perhaps add an option to some interface and structs for bitbucket, github and cloud source to indicate whether it supports build status? Seems nicer than comparing something to 'magic' strings.

Reply:
Makes sense, will do :+1:

Reply:
Actually looks like I would have to change the ci contracts Build and Pipeline structs for this. Do we really want that?

If not, alternatives I see are:
- Make this check for each repoSource. Which will make it a little bit prettier but still will be comparing to 'magic' strings;
- Create an extension `estafette-extension-source-status`. This could be just a dummy or even publish a message to a pubsub topic.